ucp/_libs/ucp_py.pyx

```diff
@@ -42,6 +42,7 @@ UCX_FILE_DESCRIPTOR = -1
 reader_added = 0
 UCP_INITIALIZED = False
 LOGGER = None
+listener_futures = []
```

Comment:
```suggestion
listener_futures = set()
```

Reply:
Should be fine either way, but in principle this should probably support O(1) removal times